.github/actions/package-builder/action.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Package Builder'
+description: 'Upload version and build package. Upload build files'
+
+inputs:
+  package:
+    description: 'The package name for build'
+    required: true
+  updateSymbols:
+    description: 'If required update Symbols version'
```

Comment:
We can add more detail as: "Updates Symbols dependency in components package.json"

Reply:
Done